BlockSettleHW/ledger/ledgerDevice.cpp

```diff
@@ -1101,7 +1101,12 @@ bool LedgerCommandThread::initDevice()
    hidDeviceInfo_.serialNumber_.toWCharArray(serNumb.get());
    serNumb.get()[hidDeviceInfo_.serialNumber_.length()] = 0x00;
    dongle_ = nullptr;
-   dongle_ = hid_open(static_cast<ushort>(Ledger::HID_VENDOR_ID), static_cast<ushort>(hidDeviceInfo_.productId_), serNumb.get());
+   try {
```

Comment:
why add try/catch here? hid_open is in C so it should not throw

Reply:
True, sorry, I miss this point. Will fix in other way